src/ru/skypro/Employee.java

```diff
@@ -0,0 +1,47 @@
+package ru.skypro;
+
+public class Employee {
+    private static int count;
+    String fullName;
```

Comment:
для группировки полей следует отделять пустой строкой статические от нестатических
```suggestion
    private static int count;
    
    private String fullName;
```